modules/dev-tools/scripts/bootstrap.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+
+const shell = require('shelljs');
+const path = require('path');
+
+// Runs the bash script and forward the arguments
+shell.exec(`${path.resolve(__dirname, './bootstrap.sh')} ${process.argv.slice(2).join(' ')}`);
```

Comment:
Would it make more sense just to have one of these scripts and pass the `.sh` script name in as an argument?

Reply:
That's what I initially tried but passing args does not seem to work with this setup :\